entity-service-impl/src/test/java/org/hypertrace/entity/service/util/DocStoreConverterTest.java

```diff
@@ -372,7 +402,7 @@ public void testStringArrayValueTypeColumnOrChain() throws JsonProcessingExcepti
             AttributeFilter.newBuilder().setOperator(Operator.AND)
                 .addChildFilter(
                     AttributeFilter.newBuilder()
-                        .setName("some_col")
+                        .setName("attributes.some_col")
```

Comment:
Assuming that in attribute service mapping all sub doc start with prefix `attribute`

Reply:
mapping of score.key attribute to sub doc path is in the entity-service itself. AttributeService just stores the scope.Key
Yes, mostly the ones in attribute service has attributes prefix, and even if it's not, it can still use it.